akeneo-design-system/src/components/Scoring/Scoring.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {Ref} from 'react';
+import styled from 'styled-components';
+import {AkeneoThemedProps, getColorForScoring, getFontSize, Score} from '../../theme';
+
+const ScoringContainer = styled.div<ScoringProps & AkeneoThemedProps>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const ScoreWrapper = styled.div<{score: Score} & ScoringProps & AkeneoThemedProps>`
```

Comment:
Personnaly, I think it would be more readable to calculate if the score is active in the functional component instead of doing it in the styled component

```suggestion
const ScoreWrapper = styled.div<{score: Score, isActive: boolean} & AkeneoThemedProps>`
```